### PR TITLE
SAMA-14 added `includes` field for `get_participants_by_cids` request

### PR DESCRIPTION
**Pull request by Oleksandr1414**
* if you want to get only the `id`, you can pass an array in the `includes` field in the form: ["id"].

### CODE REVIEW

File: app/controllers/conversations.js [hunk 1]
```diff
@@ -432,7 +432,7 @@ class ConversationsController extends BaseController {
       {
         _id: { $in: usersIds.filter((el, i) => usersIds.indexOf(el) === i) },
       },
-      ["_id", "first_name", "last_name", "login", "email", "phone"],
+      includes || ["_id", "first_name", "last_name", "login", "email", "phone"],
```
Comment (IgorKhomenko) [inline app/controllers/conversations.js:435]: `includes ` will have `["id"]` value. Should we convert it to `["_id"]` ?

**Review by IgorKhomenko — APPROVED** [review]

**Review by ccvlad — APPROVED** [review]
LGFM

File: app/controllers/users.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ class UsersController extends BaseController {
       });
     }
 
-    reqData["recent_activity"] = Date.now() / 1000;
+    reqData["recent_activity"] = Math.round(Date.now() / 1000);
```
Comment (DSuvoZera) [inline app/controllers/users.js:45]: yeah...)))

**Review by DSuvoZera — APPROVED** [review]

File: app/controllers/conversations.js (hunk 1 above)
Comment (Oleksandr1414) [reply app/controllers/conversations.js:435]: You don't have to convert, it won't affect the result. But I will add the following to increase reliability in the future.

**Pull request merged**